server/sandbox_run_linux.go

```diff
@@ -427,8 +427,9 @@ func (s *Server) runPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 	for idx := range cleanupFuncs {
 		defer func(currentFunc int) {
 			if err != nil {
-				if err2 := cleanupFuncs[currentFunc](); err != nil {
-					log.Debugf(ctx, err2.Error())
+				log.Debugf(ctx, "%v", err)
```

Comment:
do we still want this debug?

Reply:
Ah no I removed it and re-used `err` to get rid of `err2`.